src/cares_wrap.cc

```diff
@@ -1266,8 +1266,14 @@ static void Initialize(Handle<Object> target,
               Integer::New(env->isolate(), AF_UNSPEC));
   target->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "AI_ADDRCONFIG"),
               Integer::New(env->isolate(), AI_ADDRCONFIG));
+#ifdef __FreeBSD__
```

Comment:
That looks plain wrong to me, it must have worked purely by accident on platforms that are a little more forgiving.  I suggest dropping this change and filing an issue for the V4MAPPED bug.